From 3f8e3a65ecb2351cff7986bb6d5b4cdb4cd9d986 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Fri, 21 Aug 2020 16:26:39 -0700
Subject: [PATCH 24/24] Revert "ASOC: SOF: Intel: hda-codec: move unused label
 to correct position"

This reverts commit 11ec0edc6408a739dffca34ebbbe921817c3b10e.

This patch does not make any sense and adds a warning in certain
configurations:

sound/soc/sof/intel/hda-codec.c:182:1: warning: unused label 'error'
[-Wunused-label]

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 sound/soc/sof/intel/hda-codec.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/sound/soc/sof/intel/hda-codec.c b/sound/soc/sof/intel/hda-codec.c
index 55811b99e47a..7a2a612f1245 100644
--- a/sound/soc/sof/intel/hda-codec.c
+++ b/sound/soc/sof/intel/hda-codec.c
@@ -178,11 +178,6 @@ static int hda_codec_probe(struct snd_sof_dev *sdev, int address,
 	}
 
 	return ret;
-
-error:
-	snd_hdac_ext_bus_device_exit(hdev);
-	return -ENOENT;
-
 #else
 	hdev = devm_kzalloc(sdev->dev, sizeof(*hdev), GFP_KERNEL);
 	if (!hdev)
-- 
2.28.0

